python/cudf/cudf/tests/test_dataframe_init_tuples.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+import numpy as np
+import pandas as pd
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+
+def test_init_with_tuples():
+
+    data = [
+        (5, "cats", "jump", np.nan),  # 3 columns
+        (2, "dogs", "dig", 7.5),  # 3 columns
+        (3, "cows", "moo", -2.1, "occasionally"),  # 4 columns
+    ]
```

Comment:
I might add an empty tuple case, or even a case where the data is completely empty tuples.

Reply:
Didn't even think about the list of empty tuple scenario. :+1: